Purchases/Public/PurchasesDelegate.swift

```diff
@@ -20,18 +20,18 @@ import StoreKit
  * or promotional product purchases.
  *
  * - Note: Delegate methods can be called at any time after the `delegate` is set, not just in response to
- *  `purchaserInfo:` calls. Ensure your app is capable of handling these calls at anytime if `delegate` is set.
+ *  `customerInfo:` calls. Ensure your app is capable of handling these calls at anytime if `delegate` is set.
  */
 @objc(RCPurchasesDelegate) public protocol PurchasesDelegate: NSObjectProtocol {
 
     /**
      * Called whenever ``Purchases`` receives updated purchaser info. This may happen periodically
      * throughout the life of the app if new information becomes available (e.g. UIApplicationDidBecomeActive).*
      * - Parameter purchases: Related ``Purchases`` object
-     * - Parameter purchaserInfo: Updated ``PurchaserInfo``
+     * - Parameter customerInfo: Updated ``CustomerInfo``
      */
-    @objc(purchases:didReceiveUpdatedPurchaserInfo:)
-    optional func purchases(_ purchases: Purchases, didReceiveUpdated purchaserInfo: PurchaserInfo)
+    @objc(purchases:didReceiveUpdatedCustomerInfo:)
+    optional func purchases(_ purchases: Purchases, didReceiveUpdated customerInfo: CustomerInfo)
```

Comment:
This change might be too subtle... as in, when we make this change, if the developer was using the previous version, I think Xcode won't complain. I could give a warning about "nearly matching" the API, but it won't be a hard error because it is `optional`
I kinda think we should rename the delegate entirely to force an error.

Reply:
maybe we can keep the old one too and mark it using unavailable + deprecated + renamed to get auto-correct?

Reply:
Duh, yeah.

Reply:
Ahh, can't do it, since the internal param  `customerInfo` isn't included as part of the declaration, we get `Invalid redeclaration of 'purchases(_:didReceiveUpdated:)'`

Sigh.

Reply:
the sadness. 
I guess we can still have the @available + rename but we'll have to rename it to something different

Reply:
Oh yeah, I hadn't thought about actually changing the API, but that's a great point. If we use `available` with rename, xcode will just do the right thing.

Reply:
✅